components/desktop/mate/mate-settings-daemon/patches/gnome-settings-daemon-06-xft.patch

```diff
@@ -0,0 +1,33 @@
+diff -up mate-settings-daemon-1.14.0/config.h.in.orig mate-settings-daemon-1.14.0/config.h.in
```

Comment:
Upon closer inspection it looks like pango provides the same things as xft2 does. so we don't need this patch.